src/Sulu/Bundle/PageBundle/Tests/Functional/Controller/PageControllerTest.php

```diff
@@ -42,7 +43,6 @@ class PageControllerTest extends SuluTestCase
 
     public function setUp(): void
     {
-        parent::setUp();
```

Comment:
Aren't we missing something because of that?

Reply:
No there is no parent implimentation and as most TestCases don't call it I removed it also from this ones to be consistent.